src/lib/connectors/repos/queries/billing-batch-charge-versions.js

```diff
@@ -0,0 +1,70 @@
+exports.createSupplementary = `
+insert into
+water.billing_batch_charge_versions (billing_batch_id, charge_version_id)
+select :billingBatchId, cv.charge_version_id
+from water.licences l
+  join water.charge_versions cv on l.licence_ref = cv.licence_ref
+where
+  l.include_in_supplementary_billing = true
+  and l.region_id = :regionId::uuid
+  and (cv.end_date is null or cv.end_date > :fromDate)
+returning *;`;
+
+exports.createAnnual = `
+    insert into water.billing_batch_charge_versions (billing_batch_id, charge_version_id)
+    select :billingBatchId, cv.charge_version_id
+    from water.licences l
+      join water.charge_versions cv on l.licence_ref = cv.licence_ref
+      join water.charge_elements ce on cv.charge_version_id = ce.charge_version_id
```

Comment:
Do you need to join the charge element table here? Is it used as a constraint?

Reply:
It ensures any charge versions won't be processed if they don't contain any charge elements, so is quite handy from that respect

Reply:
I think here it ensures we only process charge versions which have charge elements